src/python/WMQuality/Emulators/EmulatedUnitTestCase.py

```diff
@@ -94,4 +96,13 @@ def setUp(self):
             self.inUseMemoryCache = self.memoryCachePatcher.start()
             self.addCleanup(self.memoryCachePatcher.stop)
 
+        if self.mockPyCondor:
+            self.condorPatcher = mock.patch('WMComponent.JobSubmitter.JobSubmitterPoller.ReqMgrAux', new=MockPyCondorAPI)
```

Comment:
WMComponent.JobSubmitter.JobSubmitterPoller.ReqMgrAux

Shouldn't this be

WMCore.Services.PyCondor.PyCondorAPI

Reply:
Yes :) Thanks!